ldms/src/ldmsd/ldmsd_stream.c

```diff
@@ -346,8 +346,11 @@ ldmsd_stream_subscribe(const char *stream_name,
 	}
 	LIST_FOREACH(cc, &s->s_c_list, c_ent) {
 		if (cc->c_cb_fn == cb_fn && cc->c_ctxt == ctxt) {
+#if 0
+			/* this is a high frequency common case */
```

Comment:
This looks like we're hiding a configuration error and it has nothing to do with this fix.